lib/sorbet-schema/version.rb

```diff
@@ -1,5 +1,5 @@
 # typed: strict
 
 module SorbetSchema
-  VERSION = "0.7.2"
+  VERSION = "0.8.0"
```

Comment:
This project uses [release-please](https://github.com/googleapis/release-please) to automate versioning and changelog generation. Go ahead and revert this and the lock change so that we can have it automatically release.

Reply:
Ahh oops. I changed it back.